test/e2e/positive.go

```diff
@@ -50,4 +65,140 @@ var _ = ginkgo.Describe("[PositiveFlow] Deployer validation", func() {
 			gomega.Expect(vo.Errors).To(gomega.BeEmpty())
 		})
 	})
+
+	ginkgo.Context("with a running cluster without any components", func() {
+		ginkgo.BeforeEach(func() {
+			err := deploy()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.AfterEach(func() {
+			err := remove()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.It("should perform overall deployment and verify all pods are running", func() {
+			ginkgo.By("checking that resource-topology-exporter pod is running")
+			mf, err := rte.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mf = mf.UpdateNamespace()
+
+			gomega.Eventually(func() error {
+				pod, err := getPodByRegex(fmt.Sprintf("%s-*", mf.DaemonSet.Name))
+				if err != nil {
+					return err
+				}
+
+				if pod.Status.Phase == corev1.PodRunning {
+					return nil
+				}
+
+				return fmt.Errorf("pod %v is not in %v state", pod.Name, corev1.PodRunning)
+			}, 1 * time.Minute, 15 * time.Second).ShouldNot(gomega.HaveOccurred())
+
+			ginkgo.By("checking that topo-aware-scheduler pod is running")
+			mfs, err := sched.GetManifests()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+			mfs = mfs.UpdateNamespace()
+
+			gomega.Eventually(func() error {
+				pod, err := getPodByRegex(fmt.Sprintf("%s-*", mfs.Deployment.Name))
+				if err != nil {
+					return err
+				}
+
+				if pod.Status.Phase == corev1.PodRunning {
+					return nil
+				}
+
+				return fmt.Errorf("pod %v is not in %v state", pod.Name, corev1.PodRunning)
+			}, 1 * time.Minute, 15 * time.Second).ShouldNot(gomega.HaveOccurred())
+
+			ginkgo.By("checking that noderesourcetopolgy has some information in it")
+			tc, err := clientutil.NewTopologyClient()
+			gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+			gomega.Eventually(func() error {
```

Comment:
Here we also need to ```Get()``` the object and play with it, so ```Eventually()``` is not perfect for that job.
What do you think?

Reply:
It's good enough for now :)